hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/util/CgroupsLCEResourcesHandler.java

```diff
@@ -122,17 +124,37 @@ void initConfig() throws IOException {
     }
 
     this.strictResourceUsageMode =
-        conf
-          .getBoolean(
-            YarnConfiguration.NM_LINUX_CONTAINER_CGROUPS_STRICT_RESOURCE_USAGE,
-            YarnConfiguration.DEFAULT_NM_LINUX_CONTAINER_CGROUPS_STRICT_RESOURCE_USAGE);
+      conf
+        .getBoolean(
+          YarnConfiguration.NM_LINUX_CONTAINER_CGROUPS_STRICT_RESOURCE_USAGE,
+          YarnConfiguration.DEFAULT_NM_LINUX_CONTAINER_CGROUPS_STRICT_RESOURCE_USAGE);
+
+    this.cappedResourceUsageMode =
```

Comment:
Instead of defining this new mode and its own coefficient property, I propose to only use the coefficient property and attach it to the strict mode with the same default value of 1. That way you can avoid line 138 to 144, line 399 and line 404 to 406.

Reply:
Updated with your feedback, thks